hera_mc/cm_utils.py

```diff
@@ -135,6 +150,29 @@ def _get_astropytime(_date, _time=0):
     return return_date
 
 
+def get_date_from_pair(d1, d2, ret='earliest'):
+    if d1 is None and d2 is None:
+        return None
+    retdat = None
```

Comment:
I feel that functions like these are much clearer if one uses `return` statements rather than having many different paths that set a `retval`-type variable.